assets/shaders/RayTracing.rgen

```diff
@@ -24,7 +24,7 @@ void main()
 	// Initialise separate random seeds for the pixel and the rays.
 	// - pixel: we want the same random seed for each pixel to get a homogeneous anti-aliasing.
 	// - ray: we want a noisy random seed, different for each pixel.
-	uint pixelRandomSeed = Camera.RandomSeed;
+	uint pixelRandomSeed = InitRandomSeed(Camera.RandomSeed,Camera.RandomSeed);
```

Comment:
I'm not sure I understand how this forces a wave32 instead of wave64. Care to explain in more details?

Reply:
I have no idea why this works. Currently there is no way to control the wavefront mode outside of compiler heuristics. I hope this changes in the future.